src/wakefs/fuse.cpp

```diff
@@ -217,16 +233,56 @@ bool run_in_fuse(fuse_args &args, int &status, std::string &result_json) {
   (void)close(STDOUT_FILENO);
   (void)close(STDERR_FILENO);
 
-  do waitpid(pid, &status, 0);
-  while (WIFSTOPPED(status));
+  bool has_timeout = (bool)args.command_timeout;
```

Comment:
Does this cast somehow figure out if the option is a some/none ?

Reply:
yeah exactly. We implement the "bool operator" here https://github.com/sifive/wake/blob/master/src/wcl/optional.h#L190

Reply:
That said, this var isn't as helpful/as needed now just I'll just delete it